### PR TITLE
chore(docs): revise headers

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

File: packages/dx-react-common/docs/guides/custom-theme.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The theme consists of two directories: `plugins` (plugins that we customize) and
 ### Components
 As we mentioned earlier, plugins render UI elements via render props. All render props have the postfix `Component`. For instance, the [Table](https://devexpress.github.io/devextreme-reactive/react/grid/docs/reference/table/) plugin contains the `tableComponent`, `headComponent`, `cellComponent`, and others.
 
-#### Handle the Props
+### Handle the Props
```
Comment (ushkal) [inline packages/dx-react-common/docs/guides/custom-theme.md]: This section belongs to the 'Components' thus should be indented

**Comment by ushkal** [comment]
LGTM

File: packages/dx-react-scheduler/docs/guides/fundamentals.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ To specify the date that should be initially displayed in the date navigator and
 
 .embedded-demo({ "path": "scheduler-basic/basic-setup", "showThemeSelector": true })
 
-## Appearance Customization
+## Customize the Appearance
```
Comment (MaximKudriavtsev) [inline packages/dx-react-scheduler/docs/guides/fundamentals.md]: We very often add this link to our customers in issues. Can we add a redirect to this new ref?

Comment (MaximKudriavtsev) [reply packages/dx-react-scheduler/docs/guides/fundamentals.md]: But in the Fundamentals section, we have only one big subsection) In this case redirect is not need

Comment (RomanTsukanov) [reply packages/dx-react-scheduler/docs/guides/fundamentals.md]: AFAIK, there's no way to redirect from an anchor link, so I'm afraid we can't do this

**Pull request merged**